Source/JavaScriptCore/parser/Parser.cpp

```diff
@@ -3281,6 +3313,9 @@ template <class TreeBuilder> TreeSourceElements Parser<LexerType>::parseClassFie
                 type = DefineFieldNode::Type::ComputedName;
                 break;
             }
+            case OPENBRACE: {
```

Comment:
What happens if `isStaticField` is false?

Reply:
It should throw a syntax error.